### ISSUE TITLE
Version Issue

### ISSUE THREAD

**Issue by hemanthkotaprolu** [amueller/mglearn#25]
Current version is not compatible with sklearn 1.2. In version 1.2 the load_boston dataset has been removed.

**Comment by lorenzilarissa** [comment]
Same error for me:

![image](https://user-images.githubusercontent.com/64791239/210802137-7a66f3af-7f36-4785-b0de-26de2537cda2.png)

**Comment by amueller** [comment]
Indeed, the book has been written with 0.24. I put a workaround into mglearn on pypi, can you try updating?
